deployment/armada-server/charts/armada-server/values.yaml

```diff
@@ -13,7 +13,7 @@ pulsar:
   armadaInit:
     # -- Pulsar image
     image:
-      repository: apachepulsar/pulsar
+      repository: richgross/pulsar
```

Comment:
This would work really well on mac but be hosed for most environments that people would deploy?

Maybe we should keep apachepulsar/pulsar and have a note that for M1 you could use this image?

Reply:
I'll check to see if he has amd64 image as well (that might be simplest)